src/errors/types.ts

```diff
@@ -86,11 +86,11 @@ export const EXCEPTION = {
     },
     20: {
         name: 'SWIVEL.ADDRESS.SET',
-        message: (e: Exception): string => `Swivel address already set.`,
+        message: (e: Exception): string => `Swivel address is already set.`,
```

Comment:
i don't see a reason to make them verbose with things like `...is,` and ` already...`. more words in an error message don't make it a better message

but you do you. no real requests here, just my 2C

Reply:
yeah, i wasn't sure about it either... 

my thinking was: when a user tries to set the swivel address and then gets an error message that reads "Swivel address set." - it almost looks like a confirmation, rather than an error...

especially when those messages hit the frontend and show up in a notification, things might get confusing?